### PR TITLE
Functional app

### PR DESCRIPTION
**Pull request by annamourasobral**
I already have some refactoring on hardcoded Todo's but It works.

### CODE REVIEW

File: src/components/MainApp.js [hunk 1]
```diff
@@ -1,6 +1,19 @@
-import { StyledMainApp } from "./styles/MainApp.styled";
+import { StyledMainApp } from './styles/MainApp.styled';
+import { useState } from 'react';
 
-export default function MainApp() {
+export default function MainApp({inputText, setInputText, todos, setTodos}) {
+    const inputTextHandler = (e) => {
+        console.log(e.target.value);
```
Comment (nowalor) [inline src/components/MainApp.js]: Would remove console.logs before pushing to main :)

File: src/components/Todo.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                return {
+                    ...item, completed: !item.completed
+                }
+            }
+            return item;
+        }))
+    }
+    return (
+        <StyledTodo>
+            <td className={`todo-item ${todo.completed ? "completed" : ""}`}>{text}</td>
```
Comment (nowalor) [inline src/components/Todo.js:21]: Another way to do conditionals is to create  a styled component and parse todo.completed as a prop. Then you can do all the logic inside the styled component which is quite nice :)

**Review by nowalor — APPROVED** [review]
Honestly just looks very nice. Good job! :)

**Comment by annamourasobral** [comment]
console.log removed;
the completed will be modified to update the state and the content of todo.

**Pull request merged**